examples/spring-boot-with-certmanager-example/README.md

```diff
@@ -0,0 +1,122 @@
+# Spring Boot with Cert-Manager
+
+The purpose of this example is to demonstrate how you can use the Cert-Manager Dekorate extension with a Spring Boot application.
+
+Apart from the Kubernetes Dekorate Starter dependency, this application is also using the Cert-Manager dekorate dependency:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>certmanager-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+## Enable HTTPS transport
+
+To enable the HTTPS transport in Spring Boot, we need to add the following properties:
+
+```
+server.port=8443
+server.ssl.enabled=true
+```
+
+Next, we need to configure the Keystore that our Spring Boot application will use to validate the user requests. In this example, we'll use a PKCS12 Keystore type:
+
+```
+server.ssl.key-store-type=PKCS12
+server.ssl.key-store=/path/to/keystore.p12
+server.ssl.key-store-password=the password
+```
+
+Where is the `keystore.p12` Keystore and what is its password? This is where Cert-Manager comes to play. Cert-Manager will generate a valid Keystore for your application to use.
+
+## Generate the Keystore
+
+In this section, we're going to generate the `keystore.p12` using the password `supersecret`.
+
+Let's start with the password first. We need to use a Secret resource with the value `supersecret` in the data key `password`:
+
+```
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pkcs12-pass
+data:
+  # "supersecret" in base64:
+  password: c3VwZXJzZWNyZXQ=
+type: Opaque
+```
+
+In Dekorate, we can include these resources in the file `src/main/resources/k8s/common.yml`. To include these resources as part of the final kubernetes.yml file, we need to configure Dekorate as follows:
+
+```
+dekorate.options.input-path=k8s
+```
+
+Let's now configure the certificate to generate the previous Keystore:
```

Comment:
The text here should be for me the beginning of the section `## Generate a selfsigned Certificate and keystore` 

"Let's now configure the certificate to generate the previous Keystore:" --> "Let's configure the different properties to request to generate a Selfsigned certificate, to create a PKCS123 keystore file and how the password to be used by the certificate manager to work with the keystore can be found using a secret:"